test/aircraft/Pilot/descendViaStar.spec.js

```diff
@@ -14,7 +14,7 @@ ava('.descendViaStar() sets the correct Mcp modes and values', (t) => {
 
     t.true(pilot._mcp.altitudeMode === 'VNAV');
     t.true(pilot._mcp.speedMode === 'VNAV');
-    t.true(pilot._mcp.altitude === 0);
+    t.true(pilot._mcp.altitude === 8000);
```

Comment:
I'd really like to see some additional test cases for the branching logic in this method:
```javascript
if (typeof nextAltitude === 'undefined') {
    nextAltitude = this._fms.getBottomAltitude();
}

if (_isNil(nextAltitude) || nextAltitude === Infinity) {
     return [false, 'unable to descend via STAR'];
}
```

I count 6 additional test cases here:
#### first `if`
- when `nextAltitude` is defined
- when `nextAltitude` is `undefined`

#### second `if`
- when `nextAltitude` `isNil`
- when `nextAltitude` is not `isNil`
- when `nextAltitude` is `Infinity`
- when `nextAltitude` is  not `Infinity`

Reply:
done! 😃